.github/workflows/e2e.yml

```
@@ -347,14 +348,49 @@ jobs:
             sleep 10
           done
 
+      # Playwright fixtures seed authorization source rows directly through Prisma. That intentionally
+      # bypasses product services, so run the real durable delivery processor at a test-only cadence
+      # rather than duplicating relationship writes in fixture code or waiting for the 5-second schedule.
+      - name: Start AuthZed CI outbox worker
+        id: authzed-worker-start
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2034:warning:14:1: attempt appears unused. Verify use (or export if used externally)

(shellcheck)